generated/ruby/test/test_errorreporting.rb

```diff
@@ -0,0 +1,101 @@
+#
+# Integration tests for vkit generated client library for Google Cloud ErrorReporting APIs.
+#
+
+PROJECT_NAME = "gapic-test".freeze
+SERVICE = "default".freeze
+VERSION = "20160920t135819".freeze
+USER_NAME = "testuser".freeze
+#
+# Integration test for ReportErrorsServiceApi.report_error_event
+#
+# Report first two error events (to be one group) and third event (to be another group)
+require "google/cloud/error_reporting/v1beta1/report_errors_service_api"
+require "google/devtools/clouderrorreporting/v1beta1/report_errors_service_pb"
```

Comment:
I am not so sure about the style of how this file is formatted. Is that because some section could be extracted through a tool or something?  If nothing is wrong, I'd recommend to:
```ruby
# Copyright header...

require '...'

PROJECT_NAME = ....

# ... actual code
```